### ISSUE TITLE
Support pin customization

### ISSUE THREAD

**Issue by David-Mulder** [Polymer/paper-slider#31]
Not sure what the best way to implement this spec, but adding the option for prefixes and suffixes in the pin would be great. Think of monetary values (prefix = $) or percentages (suffix = %) for example.

**Comment by FdezI** [comment]
It would be great to use "datalist" with it. I would like to show text instead of numbers. Maybe to could set a function to call for every value, to use like: <paper-slider text=[[getTextFor(index)]]> </paper-slider>

**Comment by yingqiaogit** [comment]
I agree with @FdezI. I need this feature too. The text could be used only for display. Or, the knobs in a slider could have text labels
